File: Source/WickedSick.Server.XamlParser/Elements/Controls/Page.cs
using WickedSick.Server.XamlParser.Elements.Controls;

namespace WickedSick.Server.XamlParser.Elements
{
    [Element("Fayde.Controls")]
    public class Page : UserControl
    {
        public static readonly PropertyDescription TitleProperty = PropertyDescription.Register("Title", typeof(string), typeof(Page));
        public string Title
        {
            get { return GetValue("Title") as string; }
            set { SetValue("Title", value); }
        }
    }
}

File: Source/Fayde.Client/Controls/Page.ts
/// <reference path="UserControl.ts" />

module Fayde.Controls {
    export class Page extends UserControl {
        static TitleProperty: DependencyProperty = DependencyProperty.Register("Title", () => String, Page);
        Title: string;

        constructor() {
            super();
        }
    }
    Fayde.RegisterType(Page, {
    	Name: "Page",
    	Namespace: "Fayde.Controls",
    	XmlNamespace: Fayde.XMLNS
    });
}